templates/api/knowledge-panels/health/ingredients/ingredients.tt.json

```diff
@@ -55,6 +55,15 @@
                 `,
             },
         },
-        [% END %]        
+        [% END %]       
+        [% IF 1 %]
```

Comment:
it should be if we have feature ingredient right ?

Reply:
ah it should be if we have an ingredients_list panel, but the condition I put before didn't work, so I just put 1. I'll change that.